.github/composite-actions/setup-php/action.yml

```diff
@@ -22,4 +22,4 @@ runs:
         php-version: ${{ inputs.version }}
         coverage: ${{ inputs.coverage }}
         tools: composer, ${{ inputs.tools }}
-        extensions: none, curl, dom, json, mbstring, opcache, openssl, simplexml, tokenizer, xml, xmlwriter, zip
+        extensions: none, ctype, curl, dom, json, mbstring, opcache, openssl, simplexml, tokenizer, xml, xmlwriter, zip
```

Comment:
```suggestion
        extensions: ctype, curl, dom, json, mbstring, opcache, openssl, simplexml, tokenizer, xml, xmlwriter, zip
```

;-)

Reply:
As far as I remember it is a trick to start from scratch and install only listed extensions. Without `none` also other extensions are installed, even these we don't need.

Reply:
It's exactly how @Wirone explained, it is quite old test here: https://github.com/werlos/github-actions-php-extensions - but shows the difference it makes.